From 4bda0cb887b568a21406c71cea8ec88aa2693587 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Fri, 15 Nov 2019 17:43:43 +0100
Subject: [PATCH 1/2] Drop useless dependencies

Signed-off-by: Marek Vasut <marex@denx.de>
---
 targets/surfaceless/surfaceless.cmake | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/targets/surfaceless/surfaceless.cmake b/targets/surfaceless/surfaceless.cmake
index f5061126b..b85ca98a6 100644
--- a/targets/surfaceless/surfaceless.cmake
+++ b/targets/surfaceless/surfaceless.cmake
@@ -34,10 +34,6 @@ endif ()
 pkg_check_modules(EGL REQUIRED egl)
 set(DEQP_EGL_LIBRARIES ${EGL_LIBRARIES})
 
-pkg_check_modules(GBM REQUIRED gbm)
-pkg_check_modules(KMS REQUIRED libkms)
-pkg_check_modules(DRM REQUIRED libdrm)
-
 include_directories(${GLES2_INCLUDE_PATH} ${GLES3_INCLUDE_PATH}
                     ${EGL_INCLUDE_DIRS} ${GBM_INCLUDE_DIRS}
                     ${KMS_INCLUDE_DIRS} ${DRM_INCLUDE_DIRS})
-- 
2.24.0.rc1

